### ISSUE TITLE
Query regarding AOPC metric and its implementation

### ISSUE THREAD

**Issue by JAEarly** [icrto/xML#1]
**First of all - great work!** 

I just have a quick question regarding the project.
I'm struggling to find your implementation of the AOPC metric in the repo. **Is it implemented elsewhere or am I blind?**

I'm interested as I'm using AOPC in one of my own pieces of work, and I've come across an issue where I was originally applying softmax to every model prediction before computing the AOPC, which gave very different results to doing the same without applying softmax first. With softmax, I was getting AOPC scores of around 0.21 (similar to your work), but without softmax the AOPC scores are as high as 13 (which is more comparable to the ranges of the work [Evaluating the Visualization of What a Deep Neural Network Has Learned](https://scholar.google.com/scholar_url?url=https://ieeexplore.ieee.org/iel7/5962385/6104215/07552539.pdf&hl=en&sa=T&oi=gsb-gga&ct=res&cd=0&d=7349251395468541741&ei=IvXWYqGyC6iUy9YPj6q_kAY&authuser=1&scisig=AAGBfm3GwIdi2g_H1WpeKj-qI2N9kEl32Q) for 10 permutations). **Did you apply softmax to each prediction prior to computing AOPC?**

Thank you very much!

**Comment by icrto** [comment]
Hi @JAEarly!

Sorry for the delay in answering, but I had to dig up the code :)
No, the implementation of the AOPC metric is not in the repo.
I did apply softmax prior to computing AOPC.

**Comment by JAEarly** [comment]
Great, thank you!
